frontend/js/index.js

```diff
@@ -7,6 +7,39 @@ const compareButton = () => {
     });
     window.location = link.substr(0,link.length-1);
 }
+
+function allow_group_select_checkboxes(checkbox_wrapper_id){
+    let lastChecked = null;
+    let checkboxes = document.querySelectorAll('#'+checkbox_wrapper_id+' input[type="checkbox"]');
+
+    for (let i=0;i<checkboxes.length;i++){
+        checkboxes[i].setAttribute('data-index',i);
+    }
+
+    for (let i=0;i<checkboxes.length;i++){
+        checkboxes[i].addEventListener("click",function(e){
+
+            if(lastChecked && e.shiftKey) {
+                let i = parseInt(lastChecked.getAttribute('data-index'));
+                let j = parseInt(this.getAttribute('data-index'));
+                let check_or_uncheck = this.checked;
+
+                let low = i; let high=j;
```

Comment:
Why are you introducing `low` and `high` here? Couldn't you continue using `i` or `j` ? Or rename them?